site/content/en/docs/tasks/run/kubeflow/trainjobs.md

```diff
@@ -0,0 +1,374 @@
+
+### Example: Custom PyTorch Training with TrainingRuntime
+
+Create a namespace-scoped TrainingRuntime:
+
+```yaml
+apiVersion: trainer.kubeflow.org/v1alpha1
+kind: TrainingRuntime
+metadata:
+  name: torch-custom
+  namespace: team-a
+spec:
+  mlPolicy:
+    numNodes: 1
+    torch:
+      numProcPerNode: auto
+  template:
+    spec:
+      replicatedJobs:
+        - name: node
+          template:
+            metadata:
+              labels:
+                trainer.kubeflow.org/trainjob-ancestor-step: trainer
+            spec:
+              template:
+                spec:
+                  containers:
+                    - name: trainer
+                      image: pytorch/pytorch:2.7.1-cuda12.8-cudnn9-runtime
+                      env:
+                        - name: CUSTOM_ENV
+                          value: "team-a-value"
+```
+
+Create a TrainJob that uses this namespace-scoped runtime:
+
+```yaml
+apiVersion: trainer.kubeflow.org/v1alpha1
+kind: TrainJob
+metadata:
+  name: pytorch-custom
+  namespace: team-a
+  labels:
+    kueue.x-k8s.io/queue-name: team-a-queue
+spec:
+  runtimeRef:
+    name: torch-custom
+    kind: TrainingRuntime
+    apiGroup: trainer.kubeflow.org
+  trainer:
+    image: docker.io/team-a/custom-training:latest
+    numNodes: 1
+    resourcesPerNode:
+      requests:
+        cpu: "2"
+        memory: "4Gi"
+```
+
+**Key Points:**
+- The TrainingRuntime is created in the same namespace as the TrainJob (`team-a`)
+- The `runtimeRef` specifies `kind: TrainingRuntime` to use the namespace-scoped runtime
+- Each namespace can have its own customized runtimes with different configurations
+
+## Using Kueue Priority
+
+You can set job priority using Kueue's PriorityClass:
+
+```yaml
+apiVersion: trainer.kubeflow.org/v1alpha1
+kind: TrainJob
+metadata:
+  name: high-priority-training
+  namespace: default
+  labels:
+    kueue.x-k8s.io/queue-name: user-queue
+    kueue.x-k8s.io/priority-class: high-priority
+spec:
+  runtimeRef:
+    name: torch-distributed
+    kind: ClusterTrainingRuntime
+  trainer:
+    image: docker.io/kubeflow/pytorch-dist-mnist-test:v1.0
+    numNodes: 2
+    resourcesPerNode:
+      requests:
+        cpu: "4"
+        memory: "8Gi"
+        nvidia.com/gpu: "1"
+```
+
+## LLM Fine-Tuning with Kueue
+
+For Large Language Model fine-tuning with TorchTune, you can use specialized ClusterTrainingRuntimes:
+
+```yaml
+apiVersion: trainer.kubeflow.org/v1alpha1
+kind: TrainJob
+metadata:
+  name: llama-finetuning
+  namespace: default
+  labels:
+    kueue.x-k8s.io/queue-name: gpu-queue
+spec:
+  runtimeRef:
+    name: torchtune-llama3.2-1b
+    kind: ClusterTrainingRuntime
+  initializer:
+    dataset:
+      storageUri: hf://tatsu-lab/alpaca
+    model:
+      storageUri: hf://meta-llama/Llama-3.2-1B-Instruct
+      accessToken: <your-hf-token>
+  trainer:
+    numNodes: 1
+    resourcesPerNode:
+      requests:
+        cpu: "8"
+        memory: "32Gi"
+        nvidia.com/gpu: "2"
+```
+
```

Comment:
This section seems very generic for all Job types supported in queue, drop unless we have something specific

Reply:
Yes, probably, we can just at the beginning link to the BuiltinTrainer guides: https://www.kubeflow.org/docs/components/trainer/user-guides/builtin-trainer/torchtune/